cli/remote_build.go

```diff
@@ -13,11 +13,13 @@ import (
 	"net/http"
 	"net/url"
 	"os"
+	"os/signal"
 	"path"
 	"regexp"
 	goruntime "runtime"
 	"strconv"
 	"strings"
+	"syscall"
```

Comment:
nit: It's preferred to use the import `"golang.org/x/sys/unix"` to indicate that this package requires Unix-specific symbols.